content/get-started/exploring-projects-on-github/following-people.md

```diff
@@ -12,11 +12,35 @@ versions:
 topics:
   - Profile
 ---
-When you follow people, you'll see their public activity in the activity view of your news feed.{% ifversion fpt %} If someone you follow stars a public repository, {% data variables.product.product_name %} may recommend the repository to you in the discovery view of your news feed.{% endif %} To follow someone, visit their profile page and click **Follow** under their profile image.
+
+## About followers on {% data variables.product.product_name %}
+
+When you follow people, you'll see their public activity in the activity view of your news feed.{% ifversion fpt %} If someone you follow stars a public repository, {% data variables.product.product_name %} may recommend the repository to you in the discovery view of your news feed.{% endif %}.
+
+You can unfollow someone if you do not wish to see their public activity on {% data variables.product.product_name %}.
+
+## Following a user on {% data variables.product.product_name %}
+
+1. Visit a user profile you want to follow by clicking on their user image.
+   
+![Username](../../../assets/images/help/profile/user-profile-image.png) 
+
+1. Click **Follow** button under their profile image.
 
 ![Follow user button](/assets/images/help/profile/follow-user-button.png)
```

Comment:
Indenting the image so that it appears within the numbered steps.
```suggestion
  ![Follow user button](/assets/images/help/profile/follow-user-button.png)
```